aspnetcore/toc.yml

```
@@ -652,74 +656,78 @@ items:
-      - name: Handle errors
-        uid: web-api/handle-errors
-      - name: Test with HttpRepl
+        - name: Overview
+          uid: web-api/index
+        - name: Tutorials
+          items:
+            - name: Create a web API with controllers
+              uid: tutorials/first-web-api
+            - name: Web API with MongoDB
+              uid: tutorials/first-mongo-app
+        - name: Swagger / OpenAPI
+          items:
+            - name: Overview
+              uid: tutorials/web-api-help-pages-using-swagger
+            - name: Get started with Swashbuckle
+              uid: tutorials/get-started-with-swashbuckle
+            - name: Get started with NSwag
+              uid: tutorials/get-started-with-nswag
+            - name: OpenAPI tools
+              uid: web-api/Microsoft.dotnet-openapi
+        - name: Action return types
+          uid: web-api/action-return-types
+        - name: Handle JSON Patch requests
+          uid: web-api/jsonpatch
+        - name: Format response data
+          uid: web-api/advanced/formatting
+        - name: Custom formatters
+          uid: web-api/advanced/custom-formatters
+        - name: Analyzers
+          uid: web-api/advanced/analyzers
+        - name: Conventions
+          uid: web-api/advanced/conventions
+        - name: Handle errors
+          uid: web-api/handle-errors
+        - name: Test with HttpRepl
+          items:
+            - name: Overview
+              uid: web-api/http-repl
+            - name: Telemetry
+              uid: web-api/http-repl/telemetry
+      - name: Minimal APIs
         items:
-          - name: Overview
-            uid: web-api/http-repl
-          - name: Telemetry
-            uid: web-api/http-repl/telemetry
-  - name: Minimal APIs
-    items:
-      - name: Minimal APIs overview
-        uid: fundamentals/minimal-apis/overview
-      - name: Minimal APIs quick reference
-        uid: fundamentals/minimal-apis
-      - name: WebApplication and WebApplicationBuilder
-        uid: fundamentals/minimal-apis/webapplication
-      - name: Route Handlers
-        uid: fundamentals/minimal-apis/route-handlers
-      - name: Parameter binding
-        uid: fundamentals/minimal-apis/parameter-binding
-      - name: Create responses
-        uid: fundamentals/minimal-apis/responses
-      - name: OpenAPI
-        uid: fundamentals/minimal-apis/openapi
-      - name: Filters
-        uid: fundamentals/minimal-apis/min-api-filters
-      - name: Unit and integration tests
-        uid: fundamentals/minimal-apis/test-min-api
-      - name: Middleware
-        uid: fundamentals/minimal-apis/middleware
-      - name: Handle errors
-        uid: fundamentals/minimal-apis/handle-errors
-      - name: Authentication and authorization
-        uid: fundamentals/minimal-apis/security
+        - name: Overview
+          uid: fundamentals/minimal-apis/overview
+        - name: "Tutorial: Create a minimal web API"
```

```suggestion
        - name: Tutorial: Create a minimal web API
```

Is there a reason this is quoted?

---

It has to be quoted because it contains a colon, which is a delimiter for YAML

---

Can we make it "Create a minimal API" and drop the "web"?